ui-components/toolbarbuttons.md

```diff
@@ -36,10 +36,10 @@ To create a custom toolbar button, define and register the button within the `se
 tinymce.init({
   selector: '#editor',
   toolbar: 'myCustomToolbarButton',
-  setup: (editor) => {
+  setup: function (editor) {
     editor.ui.registry.addButton('myCustomToolbarButton', {
       text: 'My Custom Button',
-      onAction: () => alert('Button clicked!')
+      onAction: function () alert('Button clicked!')
```

Comment:
This won't run, it needs braces. eg:
```
onAction: function () {
  alert('Button clicked!')
}
```

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb